tests/admin_views/tests.py

```diff
@@ -1361,10 +1362,12 @@ def test_pwd_change_custom_template(self):
         self.assertEqual(response.status_code, 200)
 
 
-def get_perm(Model, perm):
-    """Return the permission object, for the Model"""
-    ct = ContentType.objects.get_for_model(Model)
-    return Permission.objects.get(content_type=ct, codename=perm)
+def get_perm(action, Model):
```

Comment:
Why this refactoring? (Touches a lot of the other tests...)

Reply:
The idea was to centralize the logic used to build the codename and do the lookup in this function. Instead, to reduce the diff, I'll simply update `ct = ContentType.objects.get_for_model(Model)` to use `for_concrete_model=False` in case someone adds a test with a proxy model.